azure-python/__main__.py

```diff
@@ -5,21 +5,27 @@
 from pulumi_azure_native import resources
 
 # Create an Azure Resource Group
-resource_group = resources.ResourceGroup('resource_group')
```

Comment:
These changes were necessary because the Windows test of this template was failing on a formatting issue.